### PR TITLE
Issue 23 deploy

### LINKED ISSUE #23: deploying pkpd app
To enable demoing to non-developers, be nice to have a test deployment.

I've used Heroku before, its pretty easy (and free!)

- https://devcenter.heroku.com/categories/working-with-django

### PR DESCRIPTION
**Pull request by martinjrobins**
Fixes #23

App is currently live on: https://pkpdapp.herokuapp.com/

### CODE REVIEW

**Review by DavAug — CHANGES_REQUESTED** [review]
@martinjrobins 
That's really cool! I went on the web page and it worked! How come that it works without merging this branch actually? 

Just one question about the secret key.

Is the app now continuously deployed on Heroku? Because that would probably mean that I have to be a little bit careful the data upload feature I'm working on right now.

File: README.md [hunk 1]
```diff
@@ -54,6 +54,9 @@ Run the server
 
 ```bash
 $ docker run -it -p 8020:8020 \
+                 -e PORT=8020 \
+                 -e DEBUG=1 \
+                 -e SECRET_KEY=aLargeRandomSecretKey \
```
[the remark this answers is not in the document]
Comment (martinjrobins) [reply README.md:59]: you setup the environment variable in the heroku dashboard, so its not stored anywhere except within your heroku account

**Comment by martinjrobins** [comment]
> @martinjrobins
> That's really cool! I went on the web page and it worked! How come that it works without merging this branch actually?
> 
> Just one question about the secret key.
> 
> Is the app now continuously deployed on Heroku? Because that would probably mean that I have to be a little bit careful the data upload feature I'm working on right now.

I manually deployed it from my local branch

no, its not continuously deployed, I can only set that up if you give heroku dashboard permissions in github (you should have a request in your email).

**Comment by DavAug** [comment]
> > @martinjrobins
> > That's really cool! I went on the web page and it worked! How come that it works without merging this branch actually?
> > Just one question about the secret key.
> > Is the app now continuously deployed on Heroku? Because that would probably mean that I have to be a little bit careful the data upload feature I'm working on right now.
> 
> I manually deployed it from my local branch
> 
> no, its not continuously deployed, I can only set that up if you give heroku dashboard permissions in github (you should have a request in your email).

Ok cool! I just granted access

**Comment by martinjrobins** [comment]
> Ok cool! I just granted access

Hmmm, I still don't have the required permissions. Heroku says I need Admin permissions on the repo, can you give me this?

**Comment by DavAug** [comment]
@martinjrobins

I gave you admin access now. Let me know whether that solves the problem.

**Review by DavAug — APPROVED** [review]

**Pull request merged**